apps/alerts/lib/alert.ex

```diff
@@ -289,7 +289,7 @@ defimpl Poison.Encoder, for: Alerts.Alert do
   end
 
   defp format_time(t) do
-    case Timex.format(t, "{YYYY}-{M}-{D} {h24}:{m}") do
+    case Timex.format(t, "{YYYY}-{0M}-{0D}T{0h24}:{0m}:{0s}{Z:}") do
```

Comment:
**suggestion(non-blocking):** Elsewhere we use `Timex.format(time, "{ISO:Extended}")` - would that work for your desired result?